code/modules/projectiles/guns/ballistic/magweapon.dm

```diff
@@ -77,15 +77,17 @@
 
 /obj/item/gun/ballistic/automatic/magrifle/pistol
 	name = "magpistol"
-	desc = "A handgun utilizing maglev technologies to propel a ferromagnetic slug to extreme velocities."
+	desc = "A handgun utilizing maglev technologies to propel a ferromagnetic slug to extreme velocities. Packs less of a punch."
 	icon_state = "magpistol"
 	w_class = WEIGHT_CLASS_NORMAL
 	slot_flags = ITEM_SLOT_BELT
 	fire_sound = 'sound/weapons/magpistol.ogg'
 	mag_type = /obj/item/ammo_box/magazine/mmag/small
 	fire_delay = 2
 	inaccuracy_modifier = 0.25
-	cell_type = /obj/item/stock_parts/cell/magnetic/pistol
+	cell_type = /obj/item/stock_parts/cell/magnetic/pistol //We also hold less ammo
+	projectile_damage_multiplier = 0.8 //20% less damage then normal
```

Comment:
20% of damage is around as much harm as before, a small bit more. this is balanced out by ammo taking a small bit more power to shoot.